### PR TITLE
Use print() function in both Python 2 and Python 3

### PR DESCRIPTION
**Pull request by cclauss**
Legacy __print__ statements are syntax errors in Python 3 but __print()__ function works as expected in both Python 2 and Python 3.

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Fix syntax errors because they can halt the runtime.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: hack/update_owners.py [hunk 1]
```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import pring_function
```
Comment (dims) [inline hack/update_owners.py]: typo `print_function`

**Comment by SataQiu** [comment]
Thanks for your contribution! @cclauss 
You need to sign the CLA first!
Also, it is better to squash the commits into one.

**Comment by cclauss** [comment]
The CLA is designed to cover modifications of significance.

Adding ( and ) to various lines of code are trivial modifications.

**Comment by neolit123** [comment]
> The CLA is designed to cover modifications of significance.
> Adding ( and ) to various lines of code are trivial modifications.

the CNCF CLA is PR blocking for this project.
you can consult with your employer whether it is fine to sign it.

**Review by eamanu — APPROVED** [review]
LGTM

**Comment by cclauss** [comment]
Please retest this.  https://identity.linuxfoundation.org/users/cclauss is up-to-date.

**Comment by cclauss** [comment]
Closing in favor of #79016 because @k8s-ci-robot will only look at the CLA once per PR.

**Comment by cclauss** [comment]
I signed it

**Comment by cclauss** [comment]
Closing.  This file has been deleted.

**Pull request closed without merging**